lib/axiom/core/event.js

```diff
@@ -45,6 +45,74 @@ export {AxiomEvent};
 export default AxiomEvent;
 
 /**
+ * The returned meta-event will fire as soon as (with opt_fireOnce) or whenever
```

Comment:
I have a hard time understanding this comment, because it does not start with a verb. Maybe something along the lines of:
```
Create an event that fires whenever any of the events passed as parameter fires.
When opt_fireOnce is true, the event fires only once, otherwise the event fires every time.
```